src/tags/models.py

```diff
@@ -7,6 +7,29 @@
 from siteconfig.models import SiteConfig
 
 
+def get_tags_from_user(user):
+    """
+    Returns all tags from quest and badges that are linked to user
+
+    Args:
+        user (User model): user model
+
+    Returns:
+        list[Tag] : list of Tag models
+    """
+    # get models through here to prevent circular imports
+    QuestSubmission = apps.get_model("quest_manager", "QuestSubmission")
+    BadgeAssertion = apps.get_model("badges", "BadgeAssertion")
+
+    quest_tags = [name for submission in QuestSubmission.objects.filter(user=user).order_by('quest__id').distinct('quest__id') 
```

Comment:
This code seems similar to the previous method you wrote (same as below lines). Would it make sense to put these into their own method and call from both instead of duplicating the code? I'm not sure, so please just take a look and do it if it makes sense.  Let's keep it DRY where we can.

Reply:
django-fied the code
![image](https://user-images.githubusercontent.com/39788517/182746221-59574234-7343-4794-b048-3840e67a7074.png)

Reply:
This is also a possibility
![image](https://user-images.githubusercontent.com/39788517/182746826-7acd713a-6815-4523-9ab5-1db7b58e5137.png)

Reply:
im going to go with the second img for now

Reply:
First! Way easier to read!